cli/src/commands/diffedit.rs

```diff
@@ -125,6 +132,6 @@ don't make any changes, then the operation will be aborted.",
             }
```

Comment:
nit: "Reparented .."?

Reply:
I'm not sure I understand what you're suggesting here, the suggestion seems to be on the "the operation will be aborted" line.

Reply:
```rust
if args.preserve_descendants {
    writeln!(formatter, "Reparented ...")
} else {
    writeln!(formatter, "Rebased ...")
}
```
(no idea why I can't add comment to unchanged line on GitHub.)

Reply:
Let me try this. I'm not sure this level of details is warranted, but it might be. I'll add this as a separate commit to test the idea for the time being.

Reply:
"reorganize"?

Reply:
Martin has suggested "rewrite" on Discord, I'll give it a try.